app/services/match/file_reader.rb

```diff
@@ -0,0 +1,42 @@
+class Match::FileReader
+  MATCH_STARTED = /^(?<started_at>\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}) - New match (?<code>\d{8}) has started$/
+  MATCH_ENDED = /^(?<ended_at>\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}) - Match (?<code>\d{8}) has ended$/
+  KILL = /^(?<killed_at>\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}) - (?<killer>\w+) killed (?<killed>\w+) using (?<weapon>\w+)$/
+  SUICIDE = /^(?<killed_at>\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}) - <WORLD> killed (?<killed>\w+) by \w+$/
+  MATCHES = Regexp.union(MATCH_STARTED, MATCH_ENDED, KILL, SUICIDE)
+  
+  def initialize(file:)
+    @file = file
+  end
+
+  def each_match
+    @file.each_line do |line|
+      next if line.blank?
```

Comment:
Add a new line after guard clauses.
```suggestion
      next if line.blank?
      
```